css-two-column-layout/styles.css

```diff
@@ -0,0 +1,157 @@
+.container {
+  max-width: 1346px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.heading-row {
+  display: flex;
+}
+
+.row-second {
+  display: flex;
+}
+
+.row-third {
+  display: flex;
+}
+
+.row-button {
+  display: flex;
+  justify-content: center;
+  margin-top: 26px;
+}
+
+.column-one-third {
```

Comment:
since this is a `two-column` layout, `column-one-half` would be a more appropriate name. Also, make sure you are setting the width explicitly, don't let flex control the size